frontend/.env.cypress.mock

```diff
@@ -0,0 +1,2 @@
+BASE_URL=http://localhost:9001
+# BASE_URL=http://localhost:4010
```

Comment:
I'll add a comment. I left it there to be able to run against the local dev env as it makes debugging breakpoints easier instead of running against a production build.